tests/test_mocks.py

```
@@ -0,0 +1,117 @@
+import pytest
+from unittest.mock import Mock, patch, MagicMock
```

Ruff (0.8.2)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)